Geometry/TrackerCommonData/bin/mixture.py

```diff
@@ -185,8 +185,12 @@ def main(argv):
     radLenFile = open(fname + ".x0","w");
     intLenFile = open(fname + ".l0","w");
 
-    listOfMixedMaterials = loadMaterialsFile(inputFile="mixed_materials.input")
-    listOfPureMaterials = loadMaterialsFile(inputFile="pure_materials.input")
+    listOfMixedMaterials = loadMaterialsFile(
+        inputFile="../data/mixed_materials.input"
```

Comment:
It would be better to use a full path, so there is no limitation on which directory the script is run from. I think this should work in Python:
`inputFile = os.path.expandvars("$CMSSW_BASE/Geometry/TrackerCommonData/data/pure_materials.input"`

I don't know how to do this in Fortran.

Reply:
You're certainly right. I don't know how to do it in Fortran either, but I'll check

Reply:
For the record: `g77` is not available on SLC7